### PR TITLE
Button

### PR DESCRIPTION
**Pull request by therajatg**
Please review my PR

### CODE REVIEW

File: styles/component-styles/button.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    column-gap: 8px;
+}
+
+.add-to-cart-button{
+    background-color: #4267B2;
+}
+
+.delete-button{
+    background-color: #D11A2A;
+}
```
Comment (devananth) [inline styles/component-styles/button.css:53]: `css variables` can be used.

File: styles/main-style.css [hunk 2]
```diff
@@ -69,14 +74,14 @@ body{
...
-button{
+/* button{
     background-color: var(--primary-color);
     border-radius: 1rem;
     padding: 1rem;
     cursor: pointer;
     color: var(--secondary-color);
     font-size: large;
-}
+} */
```
Comment (devananth) [inline styles/main-style.css]: you can remove the commented code

**Review by devananth — CHANGES_REQUESTED** [review]
Suggested few changes.

And one suggestion , the `hover state` is not functioning properly for `button-text` , `button-icon`. Have look at it

File: docs/button.html [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+                <li><a href="navigation.html">Navigation</a></li>
+                <li><a href="modal.html">Modal</a></li>
+                <li><a href="rating.html">Rating</a></li>
+                <li><a href="toast.html">Toast/Notify/Snackbar</a></li>
+                <li><a href="simplified-grid.html">Simplified Grid</a></li>
+            </ul>
+        </div>
+        <div class="description">
+            <h3>Button</h3>
+            <h4>Buttons allow users to take actions, and make choices, with a single tap. Buttons are a quite frequent occurance in most websites  such as in e-commerce sites for buying an item or adding an item to the cart, in social media sites to follow or connect with someone or as call-to-action element in most fo the sites. Various types of buttons are as follows: </h4>
```
Comment (ttahm3d) [inline docs/button.html]: Do you really want thr text to be bold? P tag would work here instead of h4

**Review by ttahm3d — COMMENTED** [review]
Looks good. Have left a comment. Please take a look.

File: styles/component-styles/button.css (hunk 1 above)
Comment (therajatg) [reply styles/component-styles/button.css:53]: Thanks for pointing it out. Used CSS variables instead.

File: styles/main-style.css (hunk 2 above)
Comment (therajatg) [reply styles/main-style.css]: Removed the commented code.

File: docs/button.html (hunk 3 above)
Comment (therajatg) [reply docs/button.html:43]: Changed to </p>

Comment (therajatg) [reply docs/button.html]: changed h4 to p

**Review by ttahm3d — APPROVED** [review]

**Pull request merged**